.github/actions/openziti-tunnel-build-action/ubuntu-20.04/Dockerfile

```diff
@@ -1,10 +1,11 @@
 ARG CMAKE_VERSION="3.22.3"
 
-FROM ubuntu:focal
+# upstream of Ubuntu Focal 20.04 LTS
+FROM debian:bullseye
```

Comment:
Debian Bullseye is the contemporary of Ubuntu Focal, but there's always the possibility that Bullseye will roll forward to a new GLIBC that is unavailable in Focal. We have the option now, or later, to switch Focal's build to the predecessor, Debian Buster, which will always have a compatible GLIBC, but may introduce other OS differences.

Reply:
what's the advantage of change the base image? it sounds like ubuntu is the safer bet?

Reply:
It allows us to use Debian MultiArch which is not available in Ubuntu (binary deps not available for foreign archs). This leads to a simpler cross-compile environment setup that's easy to express in a Dockerfile while remaining compatible with Ubuntu.